admin-ui/src/app/features/download-card/download-card/download-card.component.ts

```diff
@@ -121,10 +122,8 @@ export class DownloadCardComponent implements OnInit {
   submit(selection) {  
     if(selection === 'true'){
       this.auditService.audit(21, 'ADM-045', {'type':'download-card','actioned':'Verify and Download'});
-      let buildURL = 'https://qa3.mosip.net/v1/admin/rid-digital-card/'+this.id+'?isAcknowledged='+true;
       this.dataStorageService
-      .downloadCard(buildURL).subscribe(
-        data => {
+      .downloadCard(this.id).subscribe(data => {
           var fileName = this.id+".pdf";
```

Comment:
The file name should be configurable with extra date time. by default we want it to be random. so <random string>_<date in iso format>

Reply:
Currently we are using the rid as the filename, if we need it to be configurable need to create one more property and access it.